Makefile

```diff
@@ -0,0 +1,158 @@
+VIRTUALENV=~/.venvs/pixua
+
+.PHONY: setup
+setup: requirements install
+
+.PHONY: requirements
+requirements: requirements_py requirements_js
+	@echo "Installed requirements"
+
+.PHONY: install
+install: requirements install_py install_js
+	@echo "Installed dependencies"
+
+.PHONY: requirements_py
+requirements_py: create_virtualenv
+	@echo "Upgrading pip and setuptools"
+	@source $(VIRTUALENV)/bin/activate;\
+	pip install --upgrade pip setuptools
+
+.PHONY: install_py
+install_py: service/requirements.txt service/test_requirements.txt
+	@echo "Installing python packages"
+	@source $(VIRTUALENV)/bin/activate;\
+	cd service;\
+	pip install --exists-action s -r requirements.txt -r test_requirements.txt
+
+.PHONY: requirements_js
+requirements_js:
+	@echo "Installing javascript npm and bower dependencies"
+	@cd web-ui;\
+	npm install;\
+	node_modules/.bin/bower install
```

Comment:
This doesn't have to be part of your pull request, but the ideal way to deal with bower install in npm is to have the `npm install` command do both npm and bower install. For that, you can move this to the package.json:

```
{
  script: {
    "install": "bower install"
  }
}
```

That way, it installs both npm and bower (yes, even if you overwrite install in the scripts) and will use the bower is just installed in node_modules. No need to have it globally or specify the path.